### PR TITLE
Fix default colors and add basic mouse support

### PR DESCRIPTION
**Pull request by wrnrlr**

### CODE REVIEW

File: src/window.rs [hunk 1]
```diff
@@ -46,6 +46,8 @@ pub enum Key {
 pub enum Event {
     Resize { width: u32, height: u32 },
     Keyboard { key: Key, pressed: bool },
+    Pointer { position: [f32;2] },
```
Comment (kvark) [inline src/window.rs]: needs a `cargo fmt` pass

Comment (kvark) [inline src/window.rs]: we should probably use `mint::Vector2D<f32>` instead for consistency with the rest of the API

**Review by kvark — COMMENTED** [review]
That looks very useful, thank you for contributing!

**Comment by wrnrlr** [comment]
I've also added support for clicks

**Review by kvark — APPROVED** [review]

**Pull request merged**